### DISCUSSION TITLE
1Panel 应用商店快速安装 Alist.

### DISCUSSION THREAD

**Discussion by wanghe-fit2cloud** [AlistGo/alist#4501] [Show and tell]
[1Panel Linux 面板](https://github.com/1Panel-dev/1Panel) ，进入应用商店，支持快速安装 Alist。
- 应用商店 - 工具，选择 Alist
![image](https://github.com/alist-org/alist/assets/46734380/0377b71f-9ab4-421d-978d-26ddc83d8487)
- 进入详情页面，点击安装，输入名称等参数后，点击确认按钮安装 Alist
![image](https://github.com/alist-org/alist/assets/46734380/a0306411-e9d6-4c6d-ab57-dc932da00740)

**Comment by vivafriend** [comment]
呵呵

**Comment by haypeng** [comment]
所以这个的账号密码在哪儿看？

**Reply by wanghe-fit2cloud** [reply]
点击左侧容器菜单，容器列表中找到 Alist 容器，操作列点击终端按钮，执行以下命令：
```shell
./alist admin
```

**Reply by haypeng** [reply]
好的  谢谢！

**Comment by woshichenghaibo** [comment]
有没有详细安装教程，比如从一台空的VPS开始？

**Reply by wanghe-fit2cloud** [reply]
1Panel 官方有提供在线安装脚本。
